Messaging.md

```diff
@@ -367,10 +395,12 @@ Note that `outcome_i` is the outcome value itself and not its hash that will be
 This type contains information provided by an oracle on an event that it will attest to.
 See [the Oracle specifications](./Oracle.md#oracle-event) for more details.
 
-#### Version 0 `oracle_event`
+**For backward compatibility reasons, this type is currently serialized as a TLV and uses u16 for collection prefix. It is expected to be changed to the new serialization format in a near future.**
```

Comment:
Nevermind, this is incorrect. I thought the data structures was `OracleEvent(OracleAnnouncement)` rather than reality which is `OracleAnnouncement(OracleEvent)` :man_facepalming: